openpype/hosts/maya/plugins/publish/validate_yeti_renderscript_callbacks.py

```diff
@@ -48,6 +48,11 @@ def get_invalid(cls, instance):
 
         yeti_loaded = cmds.pluginInfo("pgYetiMaya", query=True, loaded=True)
 
+        yeti_nodes = cmds.ls(type="pgYetiMaya")
+        if not yeti_nodes:
+            cls.log.info("No yeti nodes found.")
+            return False
```

Comment:
If the yeti callback is set in your scene but yeti is NOT loaded then with this change the scene will not be invalidated even though I'd say the callback should be removed/cleaned up.

Additionally, this will always log a warning now in Maya's script editor if Yeti is not loaded because it'll warn that node type `pgYetiMaya` does not exist.

```suggestion
        if not yeti_loaded or not cmds.ls(type="pgYetiMaya"):
            cls.log.info("No yeti nodes found.")
            return False
```
This would be the same behavior as you had written but without the warning.
However, with this added code the `if not loaded` check below would never be used.

I personally feel we should be cleaning yeti callbacks from a scene that does not have yeti loaded.

Reply:
> I personally feel we should be cleaning yeti callbacks from a scene that does not have yeti loaded.

Think that might be beyond the scope of this PR cause we might get into issues with how to differentiate between yeti callbacks and others.

Reply:
Actually that logic is already in the validator. :)
This added code however breaks it because it never continues if the plugin is unloaded (and thus there are no yeti nodes)

Reply:
I think [this suggestion should work](https://github.com/ynput/OpenPype/pull/4554/files#r1140126733) to solve the issue for now.